src/Contacts.ts

```diff
@@ -191,16 +209,14 @@ export class Contacts {
   }
 
   async allow(addresses: string[]) {
-    await ConsentList.publish(
-      addresses.map((address) => this.consentList.allow(address)),
-      this.client
+    await this.consentList.publish(
+      addresses.map((address) => this.consentList.allow(address))
```

Comment:
I know this was the case before, but what happens if the publish fails. It seems like the entries will already be allowed locally.

Reply:
great callout! i just pushed a commit to address this. entries will now be updated after a successful publish.